api-extractor/src/TypeScriptHelpers.ts

```diff
@@ -36,9 +56,43 @@ export default class TypeScriptHelpers {
   /**
    * Returns the JSDoc comments associated with the specified node, if any.
    */
-  public static getJsDocComments(node: ts.Node, sourceFile: ts.SourceFile): ts.CommentRange[] {
+  public static getJsDocComments(node: ts.Node,
+    sourceFile: ts.SourceFile,
+    errorLogger: (message: string) => void): string {
+    let jsDoc: string = '';
     // tslint:disable-next-line:no-any
-    return (ts as any).getJsDocComments(node, sourceFile);
+    if ((node as any).jsDoc && (node as any).jsDoc.length > 0) {
+      // tslint:disable-next-line:no-any
+      for (let i: number = 0; i <  (node as any).jsDoc.length; i++) {
+        // tslint:disable-next-line:no-any
+        const jsDocFullText: string = (node as any).jsDoc[i].getText();
+        const jsDocLines: string[] = jsDocFullText.split(TypeScriptHelpers.newLineRegEx);
+        const jsDocStartSeqExists: boolean = TypeScriptHelpers.jsDocStartRegEx.test(jsDocLines[0]);
+        const jsDocEndSeqExists: boolean = TypeScriptHelpers.jsDocEndRegEx.test(jsDocLines[jsDocLines.length - 1]);
+        if (!(jsDocStartSeqExists && jsDocEndSeqExists)) {
+          errorLogger('JsDoc comment must begin with \"/**\" sequence and end with \"*/\" sequence.');
+          return '';
+        }
+
+        // Remove '/**'
+        jsDocLines[0] = jsDocLines[0].replace(TypeScriptHelpers.jsDocStartRegEx, '');
+        // Remove '*/'
+        jsDocLines[jsDocLines.length - 1] = jsDocLines[jsDocLines.length - 1].replace(
+          TypeScriptHelpers.jsDocEndRegEx,
+          '');
+
+        // Remove the leading '*' from any intermediate lines
```

Comment:
Maybe include a unit test case for degenerate comments like this:

```typescript
/** This
is
* apparently
legal. */
```
Apparently they are supported by the compiler.